docs/tutorials/backends/t2star_backend.py

```diff
@@ -0,0 +1,128 @@
+import numpy as np
```

Comment:
`t2star_beckend.py` should be located with the other mock backends in `qiskit_experiments/test`. It's temporary anyway because it will be replaced by Ethan's backend.

Reply:
Done.